plugins/header_rewrite/conditions.cc

```diff
@@ -316,12 +314,12 @@ ConditionQuery::eval(const Resources &res)
 {
   std::string s;
 
+  TSDebug(PLUGIN_NAME, "Evaluating QUERY()");
   if (NULL == res._rri) {
-    TSDebug(PLUGIN_NAME, "QUERY requires remap initialization! Evaluating to false!");
+    TSDebug(PLUGIN_NAME, "\tQUERY requires remap initialization! Evaluating to false!");
```

Comment:
Maybe, but that might be a different bug to fix honestly. File a Jira? We shouldn't error out here I don't think.